JS/main.js

```diff
@@ -37,17 +36,16 @@ function addBooks(item) {
     title: bookTitle.value,
     author: bookAuthor.value,
   });
-  localStorage.setItem("CollectedBooks", JSON.stringify(books));
-  bookTitle.value = "";
-  bookAuthor.value = "";
+
+  localStorage.setItem('CollectedBooks', JSON.stringify(books));
+  bookTitle.value = '';
+  bookAuthor.value = '';
```

Comment:
- [ ] No data is still preserved in our browser memory. When I reload the page, all the pages disappear. Please let's considered working on this again and get closer to approval.

Reply:
Please specify if the information should be displayed on the page on reload or it is just required to store in local Storage.
Because when I check it out on my device, it does store in local Storage.